docs/conf.py

```diff
@@ -138,19 +142,20 @@
 # Add any paths that contain custom themes here, relative to this directory.
 # To use a different custom theme, add the directory containing the theme.
 #html_theme_path = []
-
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes. To override the custom theme, set this to the
 # name of a builtin theme or the name of a custom theme in html_theme_path.
-#html_theme = None
+html_theme = 'bootstrap-astropy'
+
+html_theme_options = {'astropy_project_menubar': True}
 
 # Custom sidebar templates, maps document names to template names.
 #html_sidebars = {}
 
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-#html_favicon = ''
+html_favicon = '_static/astropy_logo.ico'
```

Comment:
This shouldn't be needed as it's in the sphinx-astropy config

Reply:
Revert back to previous state